components/automate-ui/src/app/pages/automate-settings/automate-settings.component.spec.ts

```diff
@@ -180,10 +180,7 @@ describe('AutomateSettingsComponent', () => {
 
     describe('handleFormActivation()', () => {
 
-      using(ALL_FORMS
-          // Service Groups are not currently uncheckable through the UI
-          .filter( form => !['serviceGroupNoHealthChecks', 'serviceGroupRemoveServices']
-          .includes(form)),
+      using(ALL_FORMS.map( form => form ),
```

Comment:
What are you trying to achieve with this no-op map call?
I suspect the `filter(identity)` pattern might be more appropriate.